cmd/captplanet/run.go

```diff
@@ -147,15 +107,33 @@ func cmdRun(cmd *cobra.Command, args []string) (err error) {
 			runCfg.Satellite.Kademlia,
 			runCfg.Satellite.Audit,
 			runCfg.Satellite.StatDB,
-			o,
-			// TODO(coyle): re-enable the checker after we determine why it is panicing
+			runCfg.Satellite.Overlay,
+			// TODO(coyle): re-enable the checker after we determine why it is erroring
 			// runCfg.Satellite.Checker,
-			runCfg.Satellite.Repairer,
+			// runCfg.Satellite.Repairer,
 			runCfg.Satellite.BwAgreement,
 			runCfg.Satellite.Web,
 		)
 	}()
 
+	// start the storagenodes
+	for i := 0; i < len(runCfg.StorageNodes); i++ {
+		identity, err := runCfg.StorageNodes[i].Identity.Load()
```

Comment:
Why don't we have the identity loading inside the goroutine too? If it takes some time then it would be better to execute it in parallel.

Reply:
I'm not sure the answer. I just moved this block down. Maybe @jtolds or @egonelbre can answer this?

Reply:
I think we can move it into goroutine, unless `-race` complains.